lib/openxml/elements/run_properties.rb

```diff
@@ -32,9 +32,12 @@ class RunProperties < OpenXml::Container
         words
       ]
 
-      def initialize(*boolean_properties, **value_properties)
+      def initialize(*boolean_properties, font_color: nil, typeface: nil, **value_properties)
         super()
 
+        self.font_color = font_color
+        self.typeface = typeface
+
```

Comment:
They are private methods and can't be called by public send. I also don't want to use send and expose too much.